src/AuthLDAP.php

```diff
@@ -3778,10 +3789,13 @@ public static function searchUser(AuthLDAP $authldap)
     /**
      * Get default ldap
      *
-     * @return integer
+     * @deprecated Only search on AuthLDAP but default state is now shared with AuthMail, use getDefaultAuth() instead
+     * @todo at the moment, I did not try to fix the usage of this method, not sure how to change it at the moment, need more work.
+     * @return integer AuthLDAP ID or 0 if not found
      */
     public static function getDefault()
     {
+        // @todo trigger deprecation ?
```

Comment:
Here is how to trigger a deprecation message in GLPI.

```suggestion
        Toolbox::deprecated('The `AuthLDAP::getDefault()` method is deprecated, use `class::method()` instead.');
```